src/main/webapp/script.js

```diff
@@ -103,3 +103,33 @@ function getTimeAgo(timeInMilli) {
   }
   return Math.floor(months / 12) + " years ago";
 }
+
+function getLyrics() {
+  document.getElementById('error-lyric').innerHTML = '';
+  // Gets value from inputs and sets them
+  var artist = document.getElementById('artist').value;
+  var song = document.getElementById('song').value;
+  var request = new XMLHttpRequest();
+  var obj;
+  // Calls lyrics.ovh API
+  request.open('GET', 'https://api.lyrics.ovh/v1/' + artist + '/' + song);
+  // Returns lyrics in JSON format
+  request.onreadystatechange = function () {
+    if (this.readyState === 4) {
+      var lyricsJSON = this.responseText;
```

Comment:
Could you potentially rename this variable to lyricsJson using the similar variable format seen above?

Reply:
fixed!